codec_specs.md

```diff
@@ -58,96 +58,159 @@ A timestamp, expressed in [@!RFC3339] that notes when support for the `Codec Map
+
+Initialisation: none
+
+### V_REAL/RV10
+
+Codec ID: V_REAL/RV10
+
+Codec Name: RealVideo 1.0 aka RealVideo 5
+
+Description: Individual slices from the Real container are combined into a single frame.
+
+Initialisation: The `Private Data` contains a `real_video_props_t` structure in Big Endian byte order as found in [librmff](https://github.com/mbunkus/mkvtoolnix/blob/master/lib/librmff/librmff.h).
+
+### V_REAL/RV20
+
+Codec ID: V_REAL/RV20
+
+Codec Name: RealVideo G2 and RealVideo G2+SVT
+
+Description: Individual slices from the Real container are combined into a single frame.
+
+Initialisation: The `Private Data` contains a `real_video_props_t` structure in Big Endian byte order as found in [librmff](https://github.com/mbunkus/mkvtoolnix/blob/master/lib/librmff/librmff.h).
+
+### V_REAL/RV30
+
+Codec ID: V_REAL/RV30
+
+Codec Name: RealVideo 8
+
+Description: Individual slices from the Real container are combined into a single frame.
+
+Initialisation: The `Private Data` contains a `real_video_props_t` structure in Big Endian byte order as found in [librmff](https://github.com/mbunkus/mkvtoolnix/blob/master/lib/librmff/librmff.h).
+
+### V_REAL/RV40
+
+Codec ID: V_REAL/RV40
+
+Codec Name: rv40 : RealVideo 9
+
+Description: Individual slices from the Real container are combined into a single frame.
+
+Initialisation: The `Private Data` contains a `real_video_props_t` structure in Big Endian byte order as found in [librmff](https://github.com/mbunkus/mkvtoolnix/blob/master/lib/librmff/librmff.h).
+
+### V_QUICKTIME
+
+Codec ID: V_QUICKTIME
+
+Codec Name: Video taken from QuickTime(TM) files
+
+Description: Several codecs as stored in QuickTime, e.g. Sorenson or Cinepak. 
+
+Initialisation: The `Private Data` contains all additional data that is stored in the 'stsd' (sample description) atom in the QuickTime file **after** the mandatory video descriptor structure (starting with the size and FourCC fields). For an explanation of the QuickTime file format read [QuickTime File Format Specification](https://developer.apple.com/library/mac/documentation/QuickTime/QTFF/QTFFPreface/qtffPreface.html).
+
+### V_THEORA
+
+Codec ID: V_THEORA
+
+Codec Name: Theora
+
+Initialisation: The `Private Data` contains the first three Theora packets in order. The lengths of the packets precedes them. The actual layout is:
+
+* Byte 1: number of distinct packets '`#p`' minus one inside the CodecPrivate block. This MUST be '2' for current (as of 2016-07-08) Theora headers.
+* Bytes 2..n: lengths of the first '`#p`' packets, coded in [Xiph-style lacing]({{site.baseurl}}/index.html#lacing). The length of the last packet is the length of the CodecPrivate block minus the lengths coded in these bytes minus one.
 * Bytes n+1..: The Theora identification header, followed by the commend header followed by the codec setup header. Those are described in the [Theora specs](http://www.theora.org/doc/Theora_I_spec.pdf).
```

Comment:
On my side, the link gives `The requested URL /doc/Theora_I_spec.pdf was not found on this server.`